modules/notify/main/index.coffee

```diff
@@ -1,33 +1,65 @@
 # utility method for setting up notifications
 setupNotification = (notification, selection) ->
-  if notification.options.icon? and notification.options.icon.length > 0
-    selection.append('div')
-      .class('hx-notification-icon-container')
-      .append('i')
-        .class('hx-notification-icon ' + notification.options.icon)
-
-  selection
-    .append('div')
-      .class('hx-notification-text')
-      .text(notification.message)
+  icon = if notification.options.icon? and notification.options.icon.length > 0
+    hx.detached('div').class('hx-notification-icon-container')
+      .add(hx.detached('i').class('hx-notification-icon ' + notification.options.icon))
+
+  content = hx.detached('div').class('hx-notification-content')
+  msg = notification.message
+
+  msgIsString = hx.isString(msg) or hx.isNumber(msg)
```

Comment:
I'm not sure if we _need_ to cope with this but this basically allows `hx.notify(1)` (something you can currently do)